solution03/src/backupDatabase.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from "mongoose";
+
+const server = "127.0.0.1:27018";
+// const server = "mongodbBackup:27018"
+const backupDatabase = "cabify_bootcamp_backup";
+
+export default mongoose.createConnection(`mongodb://${server}/${backupDatabase}`, {
```

Comment:
Try to combine `backupDatabase` and `database` into a single file and expose only one function with a parameter that will be the name of the database